aries-site/src/components/content/MarkdownComponents.js

```diff
@@ -72,8 +83,32 @@ export const components = {
       fill="horizontal"
     />
   ),
+  table: props => (
+    <Box align="start" width={{ max: 'xlarge' }}>
+      <Box background="background-front" pad="medium" round="xxsmall">
+        <Table {...props} />
+      </Box>
+    </Box>
+  ),
+  tbody: props => <TableBody {...props} />,
+  thead: props => <TableHeader {...props} />,
+  td: props => <TableCell verticalAlign="top" {...props} />,
+  th: props => (
+    <TableCell
+      scope="col"
+      pad={{ horizontal: 'small', vertical: 'xsmall' }}
+      {...props}
+    >
+      <Text weight="bold">{props.children}</Text>
```

Comment:
```suggestion
      <Text color="text-strong" weight="bold">{props.children}</Text>
```
to align with our table styling

Reply:
good call

Reply:
done